packages/ag-charts-website/src/features/example-runner/framework-templates/lib/ExampleStyle.tsx

```diff
@@ -50,26 +51,31 @@ export const ExampleStyle = ({
             border-radius: 6px;
             height: 36px;
             color: var(--button-text-color);
-            background: var(--button-background-color);
+            background-color: var(--button-background-color);
             cursor: pointer;
             display: inline-block;
             font: inherit;
             font-size: 14px;
             font-weight: 500;
             letter-spacing: 0.01em;
-            text-align: center;
-            text-align-last: center;
             padding: 0.375em 1em 0.5em;
             white-space: nowrap;
             transition: background-color 0.25s ease-in-out;
         }
 
         .toolbar :is(button, select):hover {
-            background: var(--button-hover-background-color);
+            background-color: var(--button-hover-background-color);
         }
 
-        .toolbar button:hover {
-            background: var(--button-hover-background-color);
+        .toolbar button {
+            text-align: center;
+        }
+
+        .toolbar select {
+            padding-right: 2.4em;
+            background-repeat: no-repeat;
+            background-position: center right 4px;
+            background-image: var(--select-chevron);
```

Comment:
Minor - but you can use shorthands for the background

Reply:
Good shout. Have refactored to use shorthand for background.